tests/test_pipelines/test_transform.py

```diff
@@ -34,6 +35,19 @@ def test_resize():
             multiscale_mode='2333')
         build_from_cfg(transform, PIPELINES)
 
+    # test assertion if both scale and scale_factor are setted
+    with pytest.raises(AssertionError):
+        results = dict(
+            img_prefix=osp.join(osp.dirname(__file__), '../data'),
+            img_info=dict(filename='color.jpg'))
+        load = dict(type='LoadImageFromFile', with_default_meta_keys=True)
```

Comment:
Outdated?

Reply:
Forgot that, I will fix it in next commit.